Cannot run tests

### Description  
Cannot run tests

### Steps to reproduce  

1. `npm i`
2. `npm test:lib`

### Result  
![capture](https://user-images.githubusercontent.com/6439482/41910490-7d190bda-7952-11e8-90ca-83547be0b98e.PNG)
Text: ERROR in node_modules/@angular-devkit/core/node_modules/rxjs/internal/symbol/observable.d.ts(4,9): error TS2687: All declarations of 'observable' must have identical modifiers.

### Expected result  
Tests should run

--- a/package.json
+++ b/package.json
@@ -37,9 +37,9 @@
     "zone.js": "^0.8.26"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "~0.6.3",
-    "@angular-devkit/build-ng-packagr": "~0.6.3",
-    "@angular-devkit/schematics": "^0.6.3",
+    "@angular-devkit/build-angular": "~0.6.8",
+    "@angular-devkit/build-ng-packagr": "~0.6.8",
+    "@angular-devkit/schematics": "^0.6.8",
     "@angular/cli": "~6.0.8",
     "@angular/compiler-cli": "^6.0.5",
     "@angular/language-service": "^6.0.5",

